beta/nncf/tensorflow/graph/converter.py

```diff
@@ -26,7 +27,9 @@
 from beta.nncf.tensorflow.layers.data_layout import get_input_channel_axis
 from nncf.common.graph.graph import NNCFGraph
 from nncf.common.graph.graph import NNCFGraphNodeType
-from nncf.common.graph.module_attributes import ConvolutionModuleAttributes
+from nncf.common.graph.graph import NNCFNode
+from nncf.common.graph.module_attributes import ConvolutionLayerAttributes
```

Comment:
What do you think about renaming nncf.common.graph.module_attributes -> nncf.common.graph.layer_attributes

Reply:
Done